alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/Destination.kt

```diff
@@ -307,4 +315,10 @@ data class Destination(
         }
         return content
     }
+
+    private fun validateDestinationUri(destinationMessage: BaseMessage, denyHostRanges: List<String>) {
```

Comment:
Any reason we didn't have this validation logic in notification project itself rather than having to duplicate it in plugins?